docs/guides/insar_product_guide.md

```diff
@@ -193,6 +193,11 @@ Coherence is estimated from the normalized interferogram and the co-registered i
 
 When the water masking option is applied, the validity mask is further amended to apply 0 values to any pixels classified as water in the water mask. In some cases, pixels over water may still meet the coherence and amplitude threshold criteria for inclusion, even though they are not valid for use during phase unwrapping. When processing scenes with extensive coverage by coastal waters or large inland waterbodies, there can be erroneous phase jumps introduced if unwrapping proceeds over water as if it were land. In such cases, choosing the option to apply the water mask can significantly improve the results. 
 
+#### Reference point
+In order to perform phase unwrapping, a reference point must be selected. The unwrapping will proceed relative to this reference point; the 2&#960 integer multiples will be applied to the wrapped phase using this pixel as the starting point.
```

Comment:
You should be able to directly use the [unicode π](https://unicode-table.com/en/03C0/) here instead of the HTML code. While the HTML code should render correctly on the web, it doesn't render in GitHub:

![pi](https://user-images.githubusercontent.com/7882693/131912099-8821ae8b-8e06-459e-974a-dcda3f8ee19f.png)

Reply:
@jhkennedy Can you make that magic happen? I tried just replacing the html pis with unicode pis, but clearly I did something wrong. I was just re-using the pi symbol that had been used by others in the rest of the document.

Reply:
@jhkennedy Nevermind! I think I got it corrected.

Reply:
Yep looks great!